File: python/7 kyu/102.py
# https://www.codewars.com/kata/557af4c6169ac832300000ba/train/python

# Passed

def remove_rotten(bag_of_fruits):
    if bag_of_fruits is None:
        return []    

    replaced_fruits = []
    for fruit in bag_of_fruits:
        replaced_fruit = fruit.replace("rotten", str()).lower()
        replaced_fruits.append(replaced_fruit)

    return replaced_fruits

output = remove_rotten(["apple","rottenBanana","apple"])
print(output)

File: nodejs/7 kyu/102.js
// https://www.codewars.com/kata/56fe97b3cc08ca00e4000dc9/train/javascript

// Passed

function sc(apple) {
  const xCoordinate = apple.findIndex((appleArray) => appleArray.includes("B"));
  const yCoordinate = apple[xCoordinate].indexOf("B");
  const coordinates = [xCoordinate, yCoordinate];
  return coordinates;
}

const output = sc([
  ["A", "A", "A", "A", "A"],
  ["A", "A", "A", "A", "A"],
  ["A", "A", "A", "A", "A"],
  ["A", "A", "A", "A", "A"],
  ["A", "B", "A", "A", "A"],
]);
console.log(output);